services/identity-service/src/vc/vc.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { SignJsonDTO } from './dtos/Sign.dto';
+import { VerifyJsonDTO } from './dtos/Verify.dto';
+import VcService from './vc.service';
+import { ApiBadRequestResponse, ApiInternalServerErrorResponse, ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+@ApiTags('VC')
+@Controller('utils')
+export class VcController {
+  constructor(private readonly VcService: VcService) { }
+
+  @ApiOperation({ summary: 'Sign an unsigned VC' })
+  @ApiOkResponse({ description: 'VC Signed' })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiInternalServerErrorResponse({ description: 'Internal Server Error' })
+  @Post('/sign')
+  sign(@Body() body: SignJsonDTO) {
+    console.log('tjis')
+    return this.VcService.sign(body.DID, body.payload);
+  }
+
+  @ApiOperation({ summary: 'Verify a signed VC' })
+  @Post('/verify')
+  verify(@Body() body: VerifyJsonDTO) {
+    return this.VcService.sign(body.DID, body.payload);
```

Comment:
why does it call `sign` function for verify?

Reply:
this has been resolved